SampleApps/FRExample/FRExample/AppDelegate.swift

```diff
@@ -23,6 +24,36 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         FRLog.setLogLevel([ .all])
         return true
     }
+    
+    func application(_ app: UIApplication, open url: URL, options: [UIApplication.OpenURLOptionsKey : Any] = [:]) -> Bool {
+        
+        var resumeURL: URL?
+        if let resumeURI = url.valueOf("resumeURI"), let thisURI = URL(string: resumeURI) {
+            resumeURL = thisURI
+        }
+        else if let _ = url.valueOf("suspendedId") {
+            resumeURL = url
+        }
+        
+        if let resumeURL = resumeURL {
+            if let window = self.window, let rootViewController = window.rootViewController {
+                var currentController = rootViewController
+                while let presentedController = currentController.presentedViewController {
+                    currentController = presentedController
+                }
+                FRSession.authenticateWithUI(resumeURL, currentController) { (token: Token?, error) in
+                    if let error = error {
+                        FRLog.e(error.localizedDescription)
+                    }
+                    else {
+                        FRLog.i("Authenticate with ResumeURI successful: \(String(describing: token))")
```

Comment:
This may be a silly question, but is the `\` intended here?

Reply:
It's the way of Swift interpolation. Basically whenever you would like to print out a variable within String other than reconstructing the string, you can use \() to inject variable.

Reply:
Interesting, never seen this type of syntax before.